ui/src/plugins/axios.js

```diff
@@ -45,7 +45,7 @@ authAxios.interceptors.response.use(
       console.log(error.config);
       router.replace({
         name: "login",
-        query: { redirect: router.currentRoute.fullPath },
+        query: { redirect: router.currentRoute.name },
```

Comment:
Hmm... Is there any reason we are switching from `fullPath` to `name`? If that is needed, we should probably change the `redirect` argument form in [Login.vue](https://github.com/corpus-christi/corpus-christi/blob/development/ui/src/pages/Login.vue#L106) as well, as it currently uses `fullPath` to handle normal redirects as well as routes like `groups/1/managers`, which `name` would be inadequate to handle. Hope that's helpful in some way!